experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -509,9 +541,10 @@ class TextFields extends StatelessWidget {
                         ),
                       ),
                     ),
+                    SizedBox(width: 10),
```

Comment:
```suggestion
                    const SizedBox(width: 10),
```

Reply:
The `const` is added before the bracket for the `children` property, so I think we don't need to add const here:)